crates/bevy_ecs/src/schedule_v3/schedule.rs

```diff
@@ -961,25 +964,27 @@ impl ScheduleGraph {
         true
     }
 
-    fn report_cycles(&self, strongly_connected_components: &[Vec<NodeId>]) {
-        let components_with_cycles = strongly_connected_components
-            .iter()
-            .filter(|scc| scc.len() > 1)
-            .cloned()
-            .collect::<Vec<_>>();
+    fn report_cycles(&self, cycles: &[Vec<NodeId>], switch: bool) {
```

Comment:
The use of `switch` here is pretty unclear to me. Can we make this more explicit?

Reply:
Yep.